04/app.js

```diff
@@ -1 +1,6 @@
-const years = [1980, 1934, 2002, 2019];
\ No newline at end of file
+const years = [1980, 1934, 2002, 2019];
+const yearsDiff = years.map(function(y)
+{
+    return 2020-y;
```

Comment:
ok, chociaż za rok już to nie zadziała :P bardziej `return (new Date()).getFullYear() - y` :)